specification/azurestackhci/AzureStackHCI.StackHCIVM.Management/tspconfig.yaml

```diff
@@ -9,6 +12,11 @@ options:
     # `arm-resource-flattening` is only used for back-compat for specs existed on July 2024. All new service spec should NOT use this flag
     arm-resource-flattening: true
     output-file: "{azure-resource-provider-folder}/{service-name}/StackHCIVM/{version-status}/{version}/stackhcivm.json"
+  "@azure-tools/typespec-java":
+    package-dir: "azure-resourcemanager-azurestackhci"
```

Comment:
Do you have confirmation that this service is `azurestackhci`?

Reply:
- https://github.com/Azure/sdk-release-request/issues/6407#issuecomment-3085426417
- https://github.com/Azure/sdk-release-request/issues/6407#issuecomment-3085430945

Reply:
See this comment: https://github.com/Azure/azure-sdk/issues/9000#issuecomment-3086436932

Service likely need to use separate namespace.